### ISSUE TITLE
Integrate about page into django-cms

### ISSUE THREAD

**Issue by bjrne** [match4everyone/match4everything#13]
A user should be able to upload own pictures and should be able to arrange them as on our current side. Maybe best with an own plugin which integrates the picture upload functionality.

- [ ] Remove our images and replace with an avatar
- [ ] Leave/create enough placeholders to facilitate easy adaptation by users who only know the basics of html

**Comment by josauder** [comment]
Would it make sense to merge this issue with https://github.com/match4everyone/match4everything/issues/25 and replace the about page completely with a CMS?

The CMS could be used to create pages such as the about page.

**Comment by Baschdl** [comment]
Yes, would make sense when we have a clear picture of how it could work with a CMS

**Comment by josauder** [comment]
So AFAIK now we have a working CMS, what is the status on this issue?

**Comment by bjrne** [comment]
One would need to figure out a way to "start/create/add" the cms pages automatically, e.g. create a placeholder page in the automatic config. But i think this would be ready to work on, right @Baschdl? More discussion can be found in #25

**Comment by Baschdl** [comment]
Yes, how we can preload certain content is a open question. It looks complicated on a first look as django-cms doesn't support `loaddata`. I will create issues and an EPIC for django-cms later.
